demo-config.json

```diff
@@ -26,6 +26,11 @@
         "store-view-name": "Default Store View",
         "website-id": 1,
         "website-name": "Main Website"
+      },
+      "plugins": {
+        "picker": {
+           "rootCategory": 2
+         }
       }
```

Comment:
Can you also update https://github.com/hlxsites/aem-boilerplate-commerce/blob/main/default-site.json (and add `all.Store` header too while you're there)

Reply:
Ok - can you take a look just to make sure it's correct?

Reply:
@hannessolo it looks fine to me - but are you also going to contribute code which uses this resource/config? Or is that in a separate branch/PR?

Reply:
That will be in https://git.corp.adobe.com/CIF/aem-commerce-picker